apps/trustlab/src/components/Footer/FooterDescription.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-env browser */
+import { Figure, Link } from "@commons-ui/next";
+import { Stack } from "@mui/material";
+import React from "react";
+
+import RichText from "@/trustlab/components/RichText";
+
+const FooterDescription = React.forwardRef(
+  function FooterDescription(props, ref) {
+    const { description, logo, sx } = props;
+
+    if (!(logo || description)) {
+      return null;
+    }
+    return (
+      <Stack alignItems={{ xs: "center", md: "flex-start" }} sx={sx} ref={ref}>
+        <Link href="/">
+          <Figure
+            ImageProps={logo}
+            sx={{
+              display: {
+                sm: "block",
+              },
+              filter: "grayscale(100%)",
+              height: { xs: "113px", md: "113px", lg: "113px" },
+              width: { xs: "251px", md: "251px", lg: "251px" },
+            }}
+          />
+        </Link>
+        <RichText
+          variant="footer"
+          typographyProps={{
```

Comment:
```suggestion
          TypographyProps={{
```
If props go to another component, we need to capitalise the name. I think this is the same standard [material-ui](https://mui.com/material-ui/migration/migrating-from-deprecated-apis/#transitionprops) uses.

PS: Yes, we need to migrate to `slots`. May be we should have an issue to track this.